v1-preparations/ESSENTIAL_GA_OBJECTS.MD

```diff
@@ -19,8 +19,8 @@ newer provider versions. We will address these while working on the given object
+| ROLE                     | 🚀  | -                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
I think spacing here is bad (the same emote should have the same spaces, so that md table is formatted right)

Reply:
Fixed. But it doesn't matter, the table is formatted correctly anyway.

Reply:
Okay, so I'll keep this in mind, then.